src/word_search_generator/config.py

```diff
@@ -20,6 +20,7 @@
     1: ("E", "S"),
     2: ("NE", "E", "SE", "S"),
     3: ("N", "NE", "E", "SE", "S", "SW", "W", "NW"),
+    4: ("N", "NE", "SE", "SW", "W", "NW"),  # hidden words
```

Comment:
Hey, why did you want to exclude "E" and "S" from the possible directions for the hidden words?

Reply:
When I was testing it for myself, I found that most of the hidden words ended up just as east and south as the normal words—not very hidden.  Perhaps it was merely an artifact of my private test words.

As a larger picture thing, I'm in the process of writing a commit that will allow for generating a puzzle with a custom list of allowed directions, rather than limiting the choices to three (or four) hard-coded numerical levels.  The levels would remain as presets, but (I have not coded this yet) a string such as `"n,ne,nw,s,w"` could be passed in and validated against `config.dir_moves` (numeric levels will continue to function as usual).

Reply:
Yeah, if you didn't change the puzzle level (to 2 or 3) then the hidden words would only go east and south since they are the only directions available at level 1 which is the default. If you changed the puzzle level you would find the hidden words going in more directions.

Also, the hidden words are getting added after the un-hidden words, so if there are a bunch of un-hidden words there will not be as much available space for the hidden words.

Reply:
The choice to add the secret words **after** all the non-hidden words is deliberate.  They're best-effort bonus additions.  The "real" words take priority.